arangod/Aql/Functions.cpp

```diff
@@ -5286,11 +5288,7 @@ AqlValue functions::IpV4ToNumber(ExpressionContext* expressionContext,
 
     struct in_addr addr;
     memset(&addr, 0, sizeof(struct in_addr));
-#if _WIN32
-    int result = InetPton(AF_INET, &buffer[0], &addr);
-#else
     int result = inet_pton(AF_INET, &buffer[0], &addr);
```

Comment:
Glad that we can simplify this code.
Is `inet_pton` guaranteed to be available in all Windows environments we are targeting?

Reply:
Yes, I checked that.